pure-pso/templates/plugin/snapshotclass.yaml

```diff
@@ -0,0 +1,11 @@
+# The snapshotclass will be applied to ose on install
+{{- if eq .Values.orchestrator.name "openshift" }}
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+   name: pure-snapshotclass
+   annotations:
+      snapshot.storage.kubernetes.io/is-default-class: "true"
```

Comment:
Check with @sdodsley if this is what we want, just to be sure. Or maybe make it configurable in values.yaml?

Reply:
oh, thanks for catching that, he did mention in the test not to make this the default